Presentation.Web/Controllers/API/V1/OrganizationUnitController.cs

```diff
@@ -121,8 +121,12 @@ public override HttpResponseMessage Patch(int id, int organizationId, JObject ob
                 {
                     //TODO: You have to be local or global admin to change parent
 
-                    var parentId = jtoken.Value<int>();
+                    if (!_orgUnitService.IsNativeKitosUnit(id))
```

Comment:
Use the application service and GET the access rights for the unit. then you don't repeat the same logic here

Reply:
it's only an error if the parent actually changed from the current version...
so in that case you probably need to load the entity from the bdatabase (not found if null) and THEN check if the incoming values are actually different from the current state before responding with an error